src/common.h

```diff
@@ -10,6 +12,15 @@
 
 double diff_in_second(struct timespec t1, struct timespec t2);
 
+static inline void dbgprintf(const char *format, ...) {
```

Comment:
How about renaming to `ddprintf`, which means dcurl debug-oriented printf?

Reply:
Agree.